UIKit_DZ/UIKit_DZ/DemonstrationViewController.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  DemonstrationViewController.swift
+//  UIKit_DZ
+//
+//  Created by Сергей on 05.07.2021.
+//  Copyright © 2021 Сергей. All rights reserved.
+//
+
+import UIKit
+
+final class DemonstrationViewController: UIViewController {
+
+    //MARK: - Visual Components
+    let mainLabel = UILabel()
+    
+    //MARK: - life Cycle VC
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        callingViewElements()
+        navigationControllerOptions()
+    }
+    
+    //MARK: - Private Methods
+   private func callingViewElements() {
+        mainLabel.frame = CGRect(x: 90, y: 293, width: 235, height: 169)
+        mainLabel.text = "На этом месте будет Ваш текст"
+        mainLabel.numberOfLines = 0
+        mainLabel.lineBreakMode = .byWordWrapping
+        mainLabel.font = .boldSystemFont(ofSize: 18)
+        view.addSubview(mainLabel)
+        
+        view.backgroundColor = UIColor.white
+    }
+    
+    private func navigationControllerOptions() {
+        title = "Ввод"
+        navigationItem.rightBarButtonItem = UIBarButtonItem(title: "Текста", style: .plain, target: self, action: #selector(callingAlert))
+    }
+}
+
+//MARK: - extension
+extension DemonstrationViewController {
```

Comment:
```suggestion
extension DemonstrationViewController {
```
нет смысла в такой конструкции так как вещь приватная и используется только в этом VC

Reply:
```suggestion
extension UIViewController {
```
а если поместить сюда к родителю то будет доступно во всех VC